tests/paste_events/paste_events.js

```diff
@@ -21,6 +21,27 @@
       this.editor.execCommand('autoid');
     },
 
+    'test it can paste a regular text node': function() {
+      // verify we haven't broken regular copy / paste
+      var bot = this.editorBot,
+        editor = bot.editor,
+        resumeAfter = bender.tools.resumeAfter,
+        startHtml = '<p>^</p>',
+        text = 'This is some text';
+
+      bot.setHtmlWithSelection(startHtml);
+
+      resumeAfter(editor, 'allIdsComplete', function() {
+        editor.execCommand('paste', text);
+
+        assert.areSame('<p>This is some text</p>', editor.getData());
```

Comment:
That Mozilla thing was scary, but makes sense.  I see that the paste data actually was untouched.  :D